### ISSUE TITLE
Add options to disable compressing into gzip/brotli

### ISSUE THREAD

**Issue by wongmjane** [ralscha/parcel-plugin-compress#3]
It would be great if there will be an option to disable a compression format.

For instance, I have no plans to ship Brotli-compressed files because it would require re-building the NGINX on my server from source just for it.

**Comment by ralscha** [comment]
With this change you can now disable a compression format by setting the option `"enabled"` to false. 
You can do that either in the file `.compressrc`

```
{
  "brotli": {
    "enabled": false
  }
}
```

or in the package.json file 

```
"compress": {
    "brotli": {
        "enabled": false
    }
},
```

**Comment by wongmjane** [comment]
Awesome! Thank you for this update 👍

**Comment by ralscha** [comment]
Published as version 1.0.6
